instance/migrations/0049_appserver_refactor2.py

```diff
@@ -0,0 +1,135 @@
+                github_admin_organization_name=old_instance.github_admin_organization_name,
+                sub_domain=old_instance.sub_domain,
+                base_domain=old_instance.base_domain,
+                active_appserver=None,
+                # Ansible:
+                configuration_source_repo_url=old_instance.ansible_source_repo_url,
+                configuration_version=old_instance.configuration_version,
+                configuration_extra_settings=old_instance.ansible_extra_settings,
+                edx_platform_repository_url=repository_url,
+                edx_platform_commit=old_instance.commit_id,
+                # openedx_release has replaced forum_version, notifier_version, etc. which were usually the same:
+                openedx_release=old_instance.forum_version,
+            )
+
+            # Create the PR record, if any:
+            if old_instance.github_pr_url:
+                watched_pr = WatchedPullRequest.objects.create(
+                    branch_name=old_instance.branch_name,
+                    ref_type=old_instance.ref_type,
+                    github_organization_name=old_instance.github_organization_name,
+                    github_repository_name=old_instance.github_repository_name,
+                    github_pr_url=old_instance.github_pr_url,
+                    instance=new_instance,
+                )
+
+            # Get the VM, if any:
+            current_vm = old_instance.server_set.order_by("id").last()
+            if current_vm:
+                # Create one AppServer:
+                new_appserver = OpenEdXAppServer.objects.create(
+                    id=old_instance.id,
+                    created=old_instance.created,
+                    modified=old_instance.modified,
+                    _status=old_instance._status,
+                    name="AppServer 1",
+                    email=old_instance.email,
+                    protocol=old_instance.protocol,
+                    configuration_source_repo_url=old_instance.ansible_source_repo_url,
+                    configuration_version=old_instance.configuration_version,
+                    configuration_extra_settings=old_instance.ansible_extra_settings,
+                    edx_platform_repository_url=repository_url,
+                    edx_platform_commit=old_instance.commit_id,
+                    # openedx_release has replaced forum_version, notifier_version, etc. which were usually the same:
+                    openedx_release=old_instance.forum_version,
+                    use_ephemeral_databases=old_instance.use_ephemeral_databases,
+                    github_admin_organization_name=old_instance.github_admin_organization_name,
+                    configuration_database_settings="",
+                    configuration_storage_settings="",
+                    configuration_settings=old_instance.ansible_settings,
+                    owner_id=new_instance.ref.pk,
+                    server=current_vm,
```

Comment:
@bradenmacdonald Nit: Separate these using comments like you did in the `OpenEdXInstance.objects.create` call above (which is super nice).